### PR TITLE
Azure pipelines: create multi stage pipeline

### PR DESCRIPTION
**Pull request by CecileRobertMichon**
Adds yaml definition for multi stage pipeline to build vhd and create sku. Reuses the other two existing pipelines as templates.

### CODE REVIEW

**Comment by jchauncey** [comment]
LGTM

File: images/capi/packer/azure/.pipelines/generate-sas.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      az account set -s ${AZURE_SUBSCRIPTION_ID}
+      ACCOUNT_KEY=$(az storage account keys list -g ${RESOURCE_GROUP_NAME} --subscription ${AZURE_SUBSCRIPTION_ID} --account-name ${STORAGE_ACCOUNT_NAME} --query '[0].value')
+      start_date=$(date +"%Y-%m-%dT00:00Z" -d "-1 day")
+      expiry_date=$(date +"%Y-%m-%dT00:00Z" -d "+1 year")
+      az storage container generate-sas --name system --permissions lr --account-name ${STORAGE_ACCOUNT_NAME} --account-key ${ACCOUNT_KEY} --start $start_date --expiry $expiry_date | tr -d '\"' | tee -a packer/azure/vhd-url.out
+  displayName: Getting OS VHD URL
+  workingDirectory: '$(system.defaultWorkingDirectory)/images/capi'
+  condition: eq(variables.CLEANUP, 'False')
+  env:
+    AZURE_CLIENT_SECRET: $(AZURE_CLIENT_SECRET)
```
Comment (devigned) [inline images/capi/packer/azure/.pipelines/generate-sas.yaml:18]: nit: new line at end

**Review by devigned — APPROVED** [review]
One nit, but otherwise lgtm

File: images/capi/packer/azure/.pipelines/generate-sas.yaml (hunk 1 above)
Comment (CecileRobertMichon) [reply images/capi/packer/azure/.pipelines/generate-sas.yaml]: added

**Review by devigned — APPROVED** [review]
lgtm

**Pull request merged**